include/slam_toolbox/slam_mapper.hpp

```diff
@@ -46,7 +46,13 @@ class SMapper
   // convert TF pose to karto pose
   karto::Pose2 toKartoPose(const tf2::Transform & pose) const;
 
-  void configure(const rclcpp::Node::SharedPtr & node);
+  template <class NodeT>
+  void configure(NodeT && node)
+  {
+    configure(node->get_node_logging_interface(), node->get_node_parameters_interface());
```

Comment:
Again, no need for multiple overrides of the same function to do this. Just use the NodeT